src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -344,3 +446,170 @@ navigator.sendBeacon(
 )
 </script>
 ```
+
+### `ACTION_QUERY_PARAMS`
+
+<p>
+
+**Type:** `{ actionName: string, actionPayload: string }`
+</p>
+
+An object containing the query parameter names used internally by Astro when handling form action submissions.
+
+When you submit a form using an action, the following query parameters are added to the URL to track the action call:
+* `actionName` - The query parameter that contains the name of the action being called
+* `actionPayload` - The query parameter that contains the serialized form data
+
+This constant can be useful when you need to clean up URLs after a form submission. For example, you might want to remove action-related query parameters during a redirect:
+
+```ts title="src/pages/api/contact.ts" "ACTION_QUERY_PARAMS"
+import type { APIRoute } from "astro";
+import { ACTION_QUERY_PARAMS } from 'astro:actions'
+
+export const GET: APIRoute = ({ params, request }) => {
+  request.url.searchParams.delete(ACTION_QUERY_PARAMS.actionName);
+  request.url.searchParams.delete(ACTION_QUERY_PARAMS.actionPayload);
+
+  return redirect(link, 303);
+};
+```
+
+## `astro:actions` types
+
+```ts
+import type {
+  ActionAPIContext,
+  ActionClient,
+  ActionErrorCode,
+  ActionReturnType,
+  SafeResult,
+ } from 'astro:actions';
+```
+
+### `ActionAPIContext`
+
+A subset of the [Astro context object](/en/reference/api-reference/). The following properties are not available: `callAction`, `getActionResult`, `props`, and `redirect`.
+
+### `ActionClient`
+
+<p>
+
+**Types:**
+* <code>(input?: any) => Promise\<<a href="#saferesult">SafeResult</a>\></code>
+* `{ queryString?: string; orThrow: (input?: any) => Promise<Awaited<TOutput>>; }`
+</p>
+
+Represents an action to be called on the client. You can use it as a function that accepts input data and returns a Promise with a [`SafeResult` object](#saferesult) containing the action result or validation errors.
+
+The following example handles error when incrementing the like count fails:
```

Comment:
```suggestion
The following example shows how you can provide error handling with an `if` statement when incrementing the like count fails:
```
I had to be careful not to say that the following code *handles the error*, because I don't think `/* handle errors */` counts! 😆